### PR TITLE
Fix deprecated implictly nullable parameter

### PR DESCRIPTION
**Pull request by datlechin**
Fix Deprecate implicitly nullable parameter types, see https://wiki.php.net/rfc/deprecate-implicitly-nullable-types

### CODE REVIEW

**Comment by bakerkretzmar** [comment]
Can you remove the `array|string|null` type declaration you added in this PR? I don't believe that's actually an issue for the deprecation thing because that parameter isn't typed at all, and adding those types would be a backwards compatibility break (even if they're accurate).

File: src/BladeRouteGenerator.php [hunk 1]
```diff
@@ -9,7 +9,7 @@ class BladeRouteGenerator
 {
     public static $generated;
 
-    public function generate($group = null, string $nonce = null): string
+    public function generate(array|string|null $group = null, ?string $nonce = null): string
```
Comment (datlechin) [inline src/BladeRouteGenerator.php]:
```suggestion
    public function generate($group = null, ?string $nonce = null): string
```

File: src/Ziggy.php [hunk 2]
```diff
@@ -24,7 +24,7 @@ class Ziggy implements JsonSerializable
     protected Collection $routes;
 
     public function __construct(
-        protected $group = null,
+        protected array|string|null $group = null,
```
Comment (datlechin) [inline src/Ziggy.php]:
```suggestion
        protected $group = null,
```

**Comment by bakerkretzmar** [comment]
Thanks.

**Pull request merged**